2.x: MP quickstart fails to start: Error: Could not find or load main class

## Environment Details
* Helidon Version: 2.6.6-SNAPSHOT
* Helidon MP
* JDK version:
java 11.0.11 2021-04-20 LTS
GraalVM 21.3.0 Java 11 CE (Java Version 11.0.13+7-jvmci-21.3-b05)

----------

## Problem Description
If you build quickstart-mp that is generated from the Helidon archetype the resulting application jar won't run:

```
java -jar target/quickstart-mp.jar
Error: Could not find or load main class me.jdipol.mp.quickstart
Caused by: java.lang.ClassNotFoundException: me.jdipol.mp.quickstart
```

This is working with Helidon 2.6.5

## Steps to reproduce

Build the archetype in the helidon-2.x branch and run:

```
helidon init --reset --reset --url file:///path/to/helidon-2.x-repo/archetypes/catalog/target/cli-data
# Select MP, quickstart and accept other defaults
mvn clean install 
java -jar target/quickstart-mp.jar
```

Note: to run the helidon CLI this way you'll want the change in PR #8463

--- a/archetypes/bare-mp/src/main/resources/pom.xml.mustache
+++ b/archetypes/bare-mp/src/main/resources/pom.xml.mustache
@@ -15,7 +15,7 @@
     <name>{{name}}</name>
 
     <properties>
-        <mainClass>{{package}}</mainClass>
+        <mainClass>{{package}}.Main</mainClass>
     </properties>
 
     <dependencies>

--- a/archetypes/database-mp/src/main/resources/pom.xml.mustache
+++ b/archetypes/database-mp/src/main/resources/pom.xml.mustache
@@ -15,9 +15,9 @@
     <version>{{version}}</version>
     <name>{{name}}</name>
 
-        <properties>
-            <mainClass>{{package}}</mainClass>
-        </properties>
+    <properties>
+        <mainClass>{{package}}.Main</mainClass>
+    </properties>
 
     <dependencies>
         <dependency>

--- a/archetypes/quickstart-mp/src/main/resources/pom.xml.mustache
+++ b/archetypes/quickstart-mp/src/main/resources/pom.xml.mustache
@@ -14,9 +14,9 @@
     <version>{{version}}</version>
     <name>{{name}}</name>
 
-        <properties>
-            <mainClass>{{package}}</mainClass>
-        </properties>
+    <properties>
+        <mainClass>{{package}}.Main</mainClass>
+    </properties>
 
     <dependencies>
         <dependency>

